app/services/api/presenters/company.rb

```diff
@@ -25,6 +25,10 @@ def summary
         )
       end
 
+      def model
```

Comment:
Since we already have here `#cp_alignment` method, maybe also add smth like `#has_cp_alignment?` so it will be more readable in the template. What do you think?